### ISSUE TITLE
Include data from the Puppet Forge

### ISSUE THREAD

**Issue by garethr** [librariesio/support#37]
The Puppet Forge is the main package repository for Puppet Modules, available at https://forge.puppetlabs.com/.

It has an API, from which you can retrieve package data as well as a feed of releases: https://forgeapi.puppetlabs.com/

It would be great to be able to view Puppet modules on Libraries.io

CC @adreyer

**Comment by andrew** [comment]
Yeah definitely, along with chef cookbooks and ansible playbooks, Libraries can then start to connect the dots between applications and system level dependencies, which for security notifications in future could be really helpful.
